Gruntfile.js

```diff
@@ -446,6 +496,16 @@ module.exports = function (grunt) {
       ]
     },
 
+    //insert into index.html
+    insert: {
+      options:{},
+       main: {
+          src: "<%= yeoman.app %>/gitVersion.htm",
```

Comment:
Add gitVersion.html to .gitignore since it is automatically generated (so that no one adds it to the repo accidentally)